client/v2/internal/testpb/query.pulsar.go

```diff
@@ -5,8 +5,8 @@ import (
 	fmt "fmt"
 	_ "github.com/cosmos/cosmos-proto"
 	runtime "github.com/cosmos/cosmos-proto/runtime"
-	v1beta11 "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
-	v1beta1 "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
+	v1beta11 "cosmossdk.io/api/cosmos/base/query/v1beta1"
```

Comment:
no clue why this is happening.

did we make such a version?

I cannot find any reference to this, anywhere else in the code.

Reply:
probably there is a name conflict

Reply:
wonder why the changes would cause this....